com.microsoft.mrtk.core/Utilities/ConfigureSpaitializationSettings.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// Automatically configure an audio source to use a mixer-based
+    /// spatializer, such as the Microsoft Spatializer
+    /// <see cref="https://github.com/microsoft/spatialaudio-unity"/>.
+    /// </summary>
+    [RequireComponent(typeof(AudioSource))]
+    public class ConfigureSpaitializationSettings : MonoBehaviour
+    {
+        private void Start()
+        {
+            AudioSource audioSource = GetComponent<AudioSource>();
+
+            audioSource.spatialize = true;
+            audioSource.spatializePostEffects = true;
+            audioSource.spatialBlend = 1f;
+
+            audioSource.rolloffMode = AudioRolloffMode.Logarithmic;
+            audioSource.minDistance = 1f;
+            audioSource.maxDistance = 45f;
```

Comment:
Seems a little odd to me to enforce these min/max distances, are we confident that users won't ever need any other rolloff numbers?

Reply:
i pulled these values from the spatializer docs. also based on what you had noticed, with the default maxDistance (100) the sound would be too quiet too soon.

i can be convinced to not force the values :smile: